apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -478,8 +483,51 @@ export class PublishAction extends BaseRushAction {
     }
   }
 
+  private _addNpmPublishHome(): void {
+    // Example: "common\config\rush\.npmrc-publish"
+    const sourceNpmrcPublishPath: string = path.join(this.rushConfiguration.commonRushConfigFolder, '.npmrc-publish');
+
+    // Example: "common\temp\publish-home"
+    const targetNpmrcPublishFolder: string = path.join(this.rushConfiguration.commonTempFolder, 'publish-home');
+
+    // Example: "common\temp\publish-home\.npmrc"
+    const targetNpmrcPublishPath: string = path.join(targetNpmrcPublishFolder, '.npmrc');
+
+    try {
+      // Check if .npmrc-publish file exists to use for publishing
+      if (FileSystem.exists(sourceNpmrcPublishPath)) {
+        // Sync "common\config\rush\.npmrc-publish" --> "common\temp\publish-home\.npmrc"
+        Utilities.createFolderWithRetry(targetNpmrcPublishFolder);
+
+        // Copy down the committed .npmrc-publish file, if there is one
+        Utilities.copyAndTrimNpmrcFile(sourceNpmrcPublishPath, targetNpmrcPublishPath);
+      } else if (FileSystem.exists(targetNpmrcPublishPath)) {
+        // If the source .npmrc-publish doesn't exist and there is one in the target, delete the one in the target
+        console.log(`Deleting ${targetNpmrcPublishPath}`);
+        FileSystem.deleteFile(targetNpmrcPublishPath);
+      }
+    } catch (e) {
+      throw new Error(`Error syncing .npmrc-publish file: ${e}`);
+    }
+  }
+
   private _addSharedNpmConfig(env: { [key: string]: string | undefined }, args: string[]): void {
+    const userConfig: string = (process.platform === 'win32') ? 'USERPROFILE' : 'HOME';
     let registry: string = '//registry.npmjs.org/';
+
+    // Example: "common\temp\publish-home"
+    const targetNpmrcPublishFolder: string = path.join(this.rushConfiguration.commonTempFolder, 'publish-home');
+
+    // Example: "common\temp\publish-home\.npmrc"
+    const targetNpmrcPublishPath: string = path.join(targetNpmrcPublishFolder, '.npmrc');
```

Comment:
These were defined in the above function. These should probably just be private instance fields.